handler/remote.go

```diff
@@ -78,8 +79,21 @@ func downloadFile(filepath string, url string) {
 func fetchRemoteImg(url string, subdir string) config.MetaFile {
 	// url is https://test.webp.sh/mypic/123.jpg?someother=200&somebugs=200
 	// How do we know if the remote img is changed? we're using hash(etag+length)
-	log.Infof("Remote Addr is %s, pinging for info...", url)
-	etag := pingURL(url)
+	var etag string
+	
+	cacheKey := subdir+":"+helper.HashString(url)
+	
+	if val, found := config.RemoteCache.Get(cacheKey); found {
+		log.Infof("Using cache for remote addr: %s", url)
+		etag = val.(string)
```

Comment:
Maybe we should check here in case something wrong happened that could crash the program?🤔 like below
```
    if etagVal, ok := val.(string); ok {
        etag = etagVal
    } else {
        // handle the case where val is not a string
    }
```